src/core/manager.c

```diff
@@ -1984,7 +1984,9 @@ static int manager_dispatch_signal_fd(sd_event_source *source, int fd, uint32_t
                         if (MANAGER_IS_SYSTEM(m)) {
                                 /* This is for compatibility with the
                                  * original sysvinit */
-                                m->exit_code = MANAGER_REEXECUTE;
+                                r = verify_run_space_and_log("Refusing to reexecute");
+                                if (r >= 0)
+                                        m->exit_code = MANAGER_REEXECUTE;
```

Comment:
Not doing anything here on ENOSPC means that the error is just ignored (other than logging), as `manager_dispatch_signal_fd()` then returns 0. This sounds plausible indeed, I just wanted to make sure it DTRT.